packages/node_modules/@ciscospark/react-redux-spark/src/component.js

```diff
@@ -27,71 +30,116 @@ const defaultProps = {
   guestToken: ''
 };
 
-class SparkComponent extends Component {
-  static listenToSparkEvents(spark, props) {
-    spark.listenToAndRun(spark, 'change:canAuthorize', () => {
-      props.updateSparkStatus({authenticated: spark.canAuthorize});
-    });
+const PLUGINS = ['authorization', 'logger', 'people', 'phone', 'rooms'];
+const INTERNAL_PLUGINS = ['conversation', 'feature', 'flag', 'mercury', 'presence', 'search', 'team'];
 
-    spark.listenToAndRun(spark, 'change:isAuthenticating', () => {
-      props.updateSparkStatus({authenticating: spark.isAuthenticating});
-    });
+export class SparkComponent extends Component {
+  constructor(props) {
+    super(props);
 
-    spark.listenToAndRun(spark.internal.device, 'change:registered', () => {
-      props.updateSparkStatus({registered: spark.internal.device.registered});
-    });
-
-    SparkComponent.setupDevice(spark, props);
+    this.loadSparkInstance = this.loadSparkInstance.bind(this);
   }
 
   componentDidMount() {
-    const {props} = this;
     const {
       accessToken,
-      guestToken
+      guestToken,
+      spark
     } = this.props;
 
-    const spark = props.spark.get('spark');
+    const sparkInstance = spark.get('spark');
```

Comment:
I like calling it `sparkInstance` to differ from the spark redux store. Good idea!